A/ADIOS2/build_tarballs.jl

```diff
@@ -15,68 +15,70 @@ sources = [
 make -j${nproc}
 make -j${nproc} install
 install_license ../Copyright.txt ../LICENSE
 """
 
 # These are the platforms we will build for by default, unless further
 # platforms are passed in on the command line
-platforms = [
-    Platform("aarch64", "linux"; libc="glibc"),
-    Platform("aarch64", "linux"; libc="musl"),
-    Platform("powerpc64le", "linux"; libc="glibc"),
-    Platform("x86_64", "linux"; libc="glibc"),
-    Platform("x86_64", "linux"; libc="musl"),
-    Platform("x86_64", "macos"),
-    Platform("x86_64", "windows"),
-
-    # These platforms fail:
-
-    # [22:03:24] /workspace/srcdir/ADIOS2-2.7.1/source/adios2/engine/ssc/SscReader.cpp:420:71: error: narrowing conversion of ‘18446744073709551613ull’ from ‘long long unsigned int’ to ‘unsigned int’ inside { } [-Wnarrowing]
-    # [22:03:24]                  m_IO.DefineVariable<T>(b.name, {adios2::LocalValueDim});       \
-    # (32-bit architectures are officially not supported.)
-    #FAIL Platform("armv7l", "linux"; libc="glibc"),
-    #FAIL Platform("i686", "linux"; libc="glibc"),
-    #TODO Platform("i686", "linux"; libc="musl"),
-    #TODO Platform("i686", "windows"),
-
-    # [22:40:43] /workspace/srcdir/ADIOS2-2.7.1/source/adios2/toolkit/profiling/taustubs/tautimer.cpp:208:31: error: ‘__NR_gettid’ was not declared in this scope
-    # [22:40:43]      mytid = (uint64_t)syscall(__NR_gettid);
-    # (Likely the respective syscall does not exist on FreeBSD;
-    # reported as <https://github.com/ornladios/ADIOS2/issues/2705>.)
-    #FAIL Platform("x86_64", "freebsd"),
-]
+#TODO platforms = [
+#TODO     Platform("aarch64", "linux"; libc="glibc"),
+#TODO     Platform("aarch64", "linux"; libc="musl"),
+#TODO     Platform("powerpc64le", "linux"; libc="glibc"),
+#TODO     Platform("x86_64", "freebsd"),
+#TODO     Platform("x86_64", "linux"; libc="glibc"),
+#TODO     Platform("x86_64", "linux"; libc="musl"),
+#TODO     Platform("x86_64", "macos"),
+#TODO     Platform("x86_64", "windows"),
+#TODO 
+#TODO     # These platforms fail:
+#TODO 
+#TODO     # [22:03:24] /workspace/srcdir/ADIOS2-2.7.1/source/adios2/engine/ssc/SscReader.cpp:420:71: error: narrowing conversion of ‘18446744073709551613ull’ from ‘long long unsigned int’ to ‘unsigned int’ inside { } [-Wnarrowing]
+#TODO     # [22:03:24]                  m_IO.DefineVariable<T>(b.name, {adios2::LocalValueDim});       \
+#TODO     # (32-bit architectures are not supported; see
+#TODO     # <https://github.com/ornladios/ADIOS2/issues/2704>.)
+#TODO     #FAIL Platform("armv7l", "linux"; libc="glibc"),
+#TODO     #FAIL Platform("i686", "linux"; libc="glibc"),
+#TODO     #TODO Platform("i686", "linux"; libc="musl"),
+#TODO     #TODO Platform("i686", "windows"),
+#TODO ]
+platforms = supported_platforms()
+# 32-bit architectures are not supported; see
+# <https://github.com/ornladios/ADIOS2/issues/2704>
+platforms = filter(p -> arch(p) ∉ ("armv6l", "armv7l", "i686"), platforms)
```

Comment:
Can also use something like
```suggestion
platforms = filter(p -> nbits(p) == 64, platforms)
```

Reply:
Thanks!